shelley-ma/shelley-ma-test/src/Test/Cardano/Ledger/ShelleyMA/Serialisation/Roundtrip.hs

```diff
@@ -45,34 +47,36 @@ data EraIndex index where
   Mary :: EraIndex (MaryEra StandardCrypto)
   Shelley :: EraIndex (ShelleyEra StandardCrypto)
   Allegra :: EraIndex (AllegraEra StandardCrypto)
-  -- Add new Era's here, like this:
-  -- Alonzo :: EraIndex (AlonzoEra StandardCrypto)
+
+-- Add new Era's here, like this:
+-- Alonzo :: EraIndex (AlonzoEra StandardCrypto)
```

Comment:
Same comment as below, this is straightforward :slightly_smiling_face:

Reply:
Yeah, I didn't add these lines, they just got reformatted when I ran the formatter :-)